app/assets/stylesheets/partials/_utilities.scss

```diff
@@ -6,7 +9,7 @@ Utility alignments
 .u-align--right {
   text-align: right;
 }
-@media screen and (min-width: 641px){
+@media screen and (min-width: #{$media-query-breakpoint}){
```

Comment:
Pretty sure you can just do `$media-query-breakpoint`

Not sure why we've done this though?